user-guide/Advanced_Functionality/Security/About_DMS_Security/Encryption_in_DataMiner.md

```diff
@@ -0,0 +1,89 @@
+---
+uid: Encryption_in_DataMiner
+---
+
+# Encryption in DataMiner
+
+## Passwords
+
+In the context of a DataMiner System, we can distinguish between two types of passwords:
+
+- User passwords
+- Device passwords
+
+### User passwords
+
+Passwords for external users in the DataMiner System are always managed by the User Directory (Active Directory, OpenLDAP,...). To verify password security requirements, we recommend users consult their respective User Directory documentation.
+
+For local DataMiner users, [Windows will store the (hashed) password](https://docs.microsoft.com/en-us/windows-server/security/kerberos/passwords-technical-overview). The complexity and renewal policies in Windows apply to local DataMiner users.
+
+The only exception to this is DataMiner Cloud users (see [Types of users](xref:Types_of_users)). These users are fully managed by DataMiner. The passwords for these users are randomly generated (17 characters) and encrypted using the Blowfish algorithm (ECB mode, 308-bit key) before being stored.
+
+### Device passwords (tokens)
+
+All password encryption in DataMiner is done through the Blowfish algorithm (ECB mode). A 308-bit key is used for the encryption.
+
+DataMiner encrypts the following secrets using this technique:
+
+- Password parameters
+- [Credential libraries](xref:Managing_predefined_sets_of_credentials_for_SNMP_authentication)
+- Database passwords
+- Element passwords
+- Replication passwords
+
+## In-Transit Encryption
+
+Here we can distinguish several communication flows:
+
+- DataMiner Cube
+- DataMiner Web Apps & API
+- Inter-DataMiner
+- Data acquisition
+- Cassandra
+- Elasticsearch
+- NATS
+
+### DataMiner Cube
+
+DataMiner Cube will communicate with the DataMiner backend over .NET Remoting. By default, this is encrypted using the Rijndael (256-bit key, CBC mode) algorithm. The encryption key is negotiated over a 2048-bit RSA-secured communication channel.
+
+### DataMiner Web Apps & API
+
+By default, the DataMiner Web Apps (Reports, Dashboards, Monitoring,...) are served over HTTP, which is unencrypted. We recommend [enabling HTTPS and disabling HTTP](xref:Setting_up_HTTPS_on_a_DMA) to ensure all traffic is encrypted.
```

Comment:
For the sake of consistency, please use "etc." instead of an ellipsis at the end of enumerations.

Reply:
"recommend" shouldn't be followed by an infinitive. Possibly rephrase as "We recommend that you ensure" or "We recommend ensuring".